#!/usr/bin/env python
import math
import os.path
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("motor_poles", double_t, 0, "", default=0.0, min=0.0)
gen.add("invert_direction", bool_t, 0, "", default=False)
gen.add("velocity_correction", double_t, 0, "", default=1.0, min=0.0)

package_name = os.path.basename(os.path.dirname(os.path.dirname(__file__)))
node_name = package_name + "_node"
cfg_name = os.path.splitext(os.path.basename(__file__))[0]
exit(gen.generate(package_name, node_name, cfg_name))
